packages/atlas/src/config/routes.ts

```diff
@@ -30,7 +30,8 @@ export const relativeRoutes = {
     channel: (id = ':id') => `channel/${id}`,
     channels: () => 'channels',
     video: (id = ':id') => `video/${id}`,
-    editMembership: () => 'membership/edit',
+    editMembership: () => 'member/edit',
```

Comment:
maybe `my-membership/edit` ? don't have too many great ideas on this

Reply:
I think this should be `member/:handle/edit` actually, but we will need to adjust the logic a bit so that if I try to go to this route for someone else's profile, I get redirected

Reply:
yeah, the issue for me is that that handle in URL seems redundant because we already know the member that's trying to edit it, and as you mention it creates the issue of having to address edge cases of people trying to edit other members profiles